### ISSUE TITLE
Change Explore Projects page to an activity feed

### ISSUE THREAD

**Issue by cmoses8626** [smartscheduling/chatto-hub#38]
Something like Facebook's home page where you can post a comment and comment on a comment. These posts serve the role of ideas for discussion, and should have a button to convert into a project - same as the New Project workflow. Threads with new comments get moved back to top of feed.

**Comment by robamaton** [comment]
I'd suggest we hit the Slack API to observe how much activity is taking place, and prioritize that way, instead of hosting comments on Chatto Hub itself.

**Comment by cmoses8626** [comment]
Agreed. Add # total comments to the project listing, then bubble those projects to the top of the Projects page.
